spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BinderFactoryConfiguration.java

```diff
@@ -114,6 +117,13 @@ public BinderTypeRegistry binderTypeRegistry(ConfigurableApplicationContext conf
 		return new DefaultBinderTypeRegistry(binderTypes);
 	}
 
+	@Bean
+	@ConditionalOnEnabledHealthIndicator("binders")
```

Comment:
Can we document this health indicator property `management.health.binders.enabled` ?